apps/frontend/src/libs/components/table/libs/constants/table-max-height.constant.ts

```diff
@@ -0,0 +1,3 @@
+const TABLE_MAX_HEIGHT = 490;
```

Comment:
Where this number comes from?

Reply:
The max width of table's body + height of header

Reply:
I think to change this to just table's body (450) and pass it to the style also